extensions/control-plane/api/management-api/asset-api/build.gradle.kts

```diff
@@ -20,8 +20,11 @@ plugins {
 
 dependencies {
     api(project(":spi:control-plane:control-plane-spi"))
+    api(project(":spi:common:json-ld-spi"))
+    implementation(project(":extensions:common:json-ld"))
     implementation(project(":extensions:common:api:api-core"))
     implementation(project(":extensions:common:api:management-api-configuration"))
+    implementation(project(":extensions:common:http:jersey-core"))
```

Comment:
I guess this is caused by tangled dependencies, it is because the `ObjectMapperProvider`? I would make it available through another module, maybe an `spi`, nothing important, we can do that later on

Reply:
i honestly don't recall why i had to put it in, but i agree that referencing `spi`s is better. i'll see what can be done.